app/views/pages/side_nav.slim

```diff
@@ -30,7 +30,11 @@
     = render 'debug'
     = m(page.body)
 
+    - if page.pages.blank?
```

Comment:
What is this doing? Is this yet another check for `children?`

Reply:
This has been updated to use `children?` instead